index.html

```diff
@@ -1,12 +1,270 @@
+      <li class="site-main-catalog-item">
+        <a class="site-main-catalog-link" href="#">
+          <img class="site-main-catalog-item-image" src="img/smart_watch.svg" alt="Умные часы">
+          <h3 class="site-main-catalog-item-title">Умные часы</h3>
+        </a>
+      </li>
+      <li class="site-main-catalog-item">
+        <a class="site-main-catalog-link" href="#">
+          <img class="site-main-catalog-item-image" src="img/_quadrocopte.svg" alt="Квадракоптер">
+          <h3 class="site-main-catalog-item-title">Квадрокоптеры</h3>
+        </a>
+      </li>
+    </ul>
+  </nav>
+
+  <section class="site-main-benefits">
+    <ul class="site-main-benefits-list">
+      <li class="site-main-benefits-item">
+        <h3 class="site-main-benefits-title visually-hidden">Доставка</h3>
+        <input class="site-main-benefits-button" type=button value="Доставка" action="">
+      </li>
+      <li class="site-main-benefits-item">
+        <h3 class="site-main-benefits-title visually-hidden">Гарантия</h3>
+        <input class="site-main-benefits-button" type=button value="Гарантия" action="">
+      </li>
+      <li class="site-main-benefits-item">
+        <h3 class="site-main-benefits-title visually-hidden">Кредит</h3>
+        <input class="site-main-benefits-button" type=button value="Кредит" action="">
+      </li>
+    </ul>
+    <div class="site-main-benefits-constructor">
+      <b class="site-main-benefits-title">Доставка</b>
+      <p class="site-main-benefits-content">Мы с удовольствием доставим ваш товар прямо
+        к вашему подъезду совершенно бесплатно! Ведь
+        мы неплохо заработаем, поднимая его на ваш этаж./p>
+      </div>
+  </section>
+
+  <section class="site-main-add-offers"></section>
+</main>
+
+
+<footer class="site-footer">
+  <section class="site-footer-about">
+    <hr class="site-footer-about">
+    <h2 class="site-footer-about-title">О нас</h2>
+    <p class="site-footer-about-content">Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.</p>
+    <p class="site-footer-about-content">Мы можем доставить ваш товар в самые отдалённые уголки России транспортными компаниями:</p>
+    <ul class="site-footer-about-list">
+      <li class="site-footer-about-item">Меловые линии</li>
+      <li class="site-footer-about-item">СДИК</li>
+      <li class="site-footer-about-item">ТоксБерри</li>
+    </ul>
+    <a class="site-footer-about-more" href="#">Подробнее о нас</a>
+  </section>
+
+  <section class="site-footer-contacts">
+    <hr class="site-footer-contacts">
+    <h2 class="site-footer-contacts-title">Контакты</h2>
+    <h3 class="site-footer-contacts-heading">Интернет-магазин «Девайс»</h3>
+    <p class="site-footer-contacts-content">Вы можете забрать товар сами, заехав в наш офис.<br>Заодно сможете проверить работоспособность покупки.</p>
+    <p class="site-footer-contacts-content">Санкт-Петербург, набережная реки Карповки, 5, литера П.<br>+7 (812) 812-12-12</p>
```

Comment:
```suggestion
    <p class="site-footer-contacts-content">Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
    +7 (812) 812-12-12
```

сделай телефон через ссылку с телефоном